conf/defaults.ini

```diff
@@ -1399,7 +1399,9 @@ plugin_catalog_url = https://grafana.com/grafana/plugins/
 plugin_catalog_hidden_plugins =
 # Log all backend requests for core and external plugins.
 log_backend_requests = false
-# Force download of public key for verifying plugin signature on startup.
+# Skip downloading the public key for verifying plugin signatures.
+skip_public_key_download = false
+# Force download of public key for verifying plugin signature on startup (only if not skipped).
 enforce_public_key_download = false
```

Comment:
Tend to agree. The convention with configuration is that boolean settings should be named so that the default value is false, however this is not always true mainly due to legacy reasons.

A couple suggestions of alternatives
```
[plugins]
# Disable downloading of the public key for verifying plugin signatures. Default false.
public_key_retrieval_disabled = false
# Force downloading of the public key for verifying plugin signature on startup, rather than each X (10?) day. If set to true, requires public_key_retrieval_disabled to be false to have any effect. Default false.
public_key_retrieval_on_startup = false
```

```
[plugins.public_key_retrieval]
# Disable downloading of the public key for verifying plugin signatures. Default false.
disabled = false
# Force downloading of the public key for verifying plugin signature on startup, rather than each X (10?) day. If set to true, requires disabled to be false to have any effect. Default false.
on_startup = false
```